### PR TITLE
add mavedb data source tool

### PR DESCRIPTION
**Pull request by PlushZ**
This tool is the tool for mave data. It is a data_source tool to bring MAVE data from MaveDB to Galaxy.

### CODE REVIEW

**Comment by PlushZ** [comment]
Can we add this tool and hide (make it available from direct link)?

**Comment by bgruening** [comment]
shed.yml file is missing :)

**Comment by PlushZ** [comment]
> shed.yml file is missing :)

it's there, in the same folder as wrapper, no?

File: tools/mave_tools/mavedb/.shed.yml [hunk 1]
```diff
@@ -0,0 +1,9 @@
+categories:
+- MAVE
```
Comment (bgruening) [inline tools/mave_tools/mavedb/.shed.yml:2]: Those needs to be ToolShed categories. https://toolshed.g2.bx.psu.edu/

File: tools/mave_tools/mavedb/mavedb.xml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0"?>
+<!--
+    If the value of 'URL_method' is 'get', the request will consist of the value of 'URL' coming back in
+    the initial response.  If value of 'URL_method' is 'post', any additional params coming back in the
+    initial response ( in addition to 'URL' ) will be encoded and appended to URL and a post will be performed.
+-->
+<tool name="MaveDB" id="mavedb" tool_type="data_source">
```
Comment (bgruening) [inline tools/mave_tools/mavedb/mavedb.xml:7]:
```suggestion
<tool id="mavedb" name="MaveDB" tool_type="data_source">
```

Comment (bgruening) [reply tools/mave_tools/mavedb/mavedb.xml:7]: @wm75 how was it with the profiles? Should we add one?

Comment (bgruening) [reply tools/mave_tools/mavedb/mavedb.xml:7]: Should we name the ID to something like mavedb_importer ?

**Review by bgruening — APPROVED** [review]

**Pull request merged**